starter-code/ShoppingListWithDetailView/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistdetailview/DetailActivity.java

```diff
@@ -0,0 +1,43 @@
+package ly.generalassemb.drewmahrt.shoppinglistdetailview;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.TextView;
+
+import org.w3c.dom.Text;
+
+public class DetailActivity extends AppCompatActivity {
+
+    public static final String IDKEY = "IDKEY";
+    public TextView mText1, mText2, mText3, mText4;
```

Comment:
Try to give your variables more descriptive names, not just append a number onto the end.